plasmapy/diagnostics/thomson.py

```diff
@@ -55,18 +55,18 @@ def spectral_density(
     is assumed to be quasi-neutral)
 
     .. math::
-        S(k,\omega) = \sum_e \frac{2\pi}{k}
-        \bigg |1 - \frac{\chi_e}{\epsilon} \bigg |^2
-        f_{e0,e} \bigg (\frac{\omega}{k} \bigg ) +
-        \sum_i \frac{2\pi Z_i}{k}
-        \bigg |\frac{\chi_e}{\epsilon} \bigg |^2 f_{i0,i}
-        \bigg ( \frac{\omega}{k} \bigg )
-
-    where :math:`\chi_e` is the electron component susceptibility of the
-    plasma and :math:`\epsilon = 1 + \sum_e \chi_e + \sum_i \chi_i` is the total
-    plasma dielectric  function (with :math:`\chi_i` being the ion component
+        S(k,ω) = \sum_e \frac{2π}{k}
+        \bigg |1 - \frac{χ_e}{ε} \bigg |^2
+        f_{e0,e} \bigg (\frac{ω}{k} \bigg ) +
+        \sum_i \frac{2π Z_i}{k}
+        \bigg |\frac{χ_e}{ε} \bigg |^2 f_{i0,i}
+        \bigg ( \frac{ω}{k} \bigg )
```

Comment:
Just noting that I want to double check that this renders nicely... [sure does!](https://plasmapy--1375.org.readthedocs.build/en/1375/api/plasmapy.diagnostics.thomson.spectral_density.html#plasmapy.diagnostics.thomson.spectral_density)